x-pack/plugins/infra/common/log_views/resolved_log_view.mock.ts

```diff
@@ -10,6 +10,13 @@ import { createStubDataView } from '@kbn/data-views-plugin/common/stubs';
 import { defaultLogViewsStaticConfig } from './defaults';
 import { ResolvedLogView, resolveLogView } from './resolved_log_view';
 import { LogViewAttributes } from './types';
+import { DataView, DataViewSpec } from '@kbn/data-views-plugin/common';
+
+const dataViewMock = {
+  id: 'log-view-data-view-mock',
+  title: 'log-view-data-view-mock-title',
+  fields: [],
+} as unknown as DataView;
```

Comment:
Nit: In the interest of not diverging the real data view interface in the future, would it make sense to use `createStubDataView()` here too? Or would that cause other problems?

Reply:
Oh, good shout. I'd added this change before I knew about `createStubDataView()` in the later commits. I'll change it.